core/src/test/resources/org/apache/calcite/test/RelOptRulesTest.xml

```diff
@@ -2416,6 +2416,24 @@ LogicalProject(X=[+($0, $1)], B=[$1], A=[$0])
         <Resource name="planAfter">
             <![CDATA[
 LogicalValues(tuples=[[{ 11, 1, 10 }, { 23, 3, 20 }]])
+]]>
+        </Resource>
+    </TestCase>
+    <TestCase name="testReduceValuesNull">
+        <Resource name="sql">
+            <![CDATA[insert into sales.dept(deptno,name) values (0, 'null')]]>
```

Comment:
Need to change this line as the test's query is already changed.

Reply:
Yes, I update the PR. thanks.